frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -106,13 +131,42 @@ const CandidateDecider: React.FC<CandidateDeciderProps> = ({ uuid }) => {
           uuid: ''
         }
       ]);
+      setIsSaved(true);
     }
   };
 
+  useEffect(() => {
+    setIsSaved(currentComment === defaultCurrentComment && currentRating === defaultCurrentRating);
```

Comment:
instead of storing this value in a state, what if we just had a variable that stored it? `currentComment, currentRating, defaultCurrentComment, defaultCurrentRating` are all state variables so they'd trigger a re-render anyways when they're changed.

In general it isn't the best pattern to store state that's derived from other states, since it requires you to keep them in sync, and can get lead to buggy behavior if they ever became out of sync